doc/iterations/2024.3/shapes/improve_resizing_support_for_list_nodes.adoc

```diff
@@ -0,0 +1,38 @@
+= Improves resizing support for list nodes
+
+== Problem
+
+Until now, when a user resizes a list node, it is not functional.
+Only the container rendering is displayed during the resize action, the sub-container being stretched during the next layout.
+And it's not possible to reduce the size of the container (blocked by the size of the sub-elements).
+Sub-container can be resized in any direction, though it is up to the container to guarantee the width.
+
+== Key Result
+
+* When a container is resized, its sub-container must retain the rendering during the entire action.
+* Sub-container must be resizable vertically only.
+
+
+== Solution
+
+* When a resize event for a list node is triggered, create corresponding event to resize sub-elements.
+* Change sub-container node resizer to only allow vertical resize.
```

Comment:
- Should the node A1 be vertically resizable?
- If I create 10 nodes in the blue node containing B, should the parent node be resized or should sibling nodes change their size?

We need to specify the various rules.

Reply:
It's now more explicit with the CSS rule.
If the blue compartment grows, it depends on the growable childs defined and the space left on the others compartment